Documentation/pt-br/README.md

```diff
@@ -0,0 +1,39 @@
+# Documentação
+
+Quick te ajuda a verificar como programas em Swift e Objective-C se comportam.
+
+Fazê-lo de forma eficaz não é apenas questão de saber como usar Quick, no entanto. Os guias nesse diretório podem te ajudar a escrever testes eficazes --não apenas usando Quick, até mesmo XCTest ou outros framework de teste.
```

Comment:
>Fazê-lo de forma eficaz não é apenas questão de saber como usar Quick, no entanto

I'd drop "no entanto" here.

>não apenas usando Quick, até mesmo XCTest

Feels like it's missing "mas" before "até mesmo"